Sources/MetadataSystem/MetadataParser.swift

```diff
@@ -77,14 +76,18 @@ public extension MetadataParser {
 extension MetadataParser {
     // swiftlint:disable:next identifier_name
     func _visit<Definition: MetadataDefinition>(definition: Definition) {
-        if StandardEmptyMetadataVisitor(parser: self)(definition) == nil {
+        if let emptyBlock = definition as? any EmptyMetadata {
```

Comment:
This is a Swift 5.6 language feature right? I think the change requires set the minimum required version in the Package.swift to swift 5.6 right?

Reply:
tbh i thought it was a Swift 5.7 thing (some of the existentials stuff was added in 5.6, and then the remaining things came with 5.7) , but regardless I will adjust the requirement